File: problem134.java
public class problem134
{

    public static long match(int p1, int p2)
    {
	int mult = (int)Math.pow(10,Math.ceil(Math.log10(p1)));
	long start = 1;
	long number = start*mult + p1;
	while (!(number % p2 == 0))
	    {
		start++;
		number = start*mult+p1;
	    }

	return number;
    }

    public static void main(String[] args)
    {
	boolean[] primes = new boolean[1000001];
	for(int i = 2; i < primes.length ; i++)
	    {
		if (!primes[i])
		    {
			for(int j = 2*i; j < primes.length ; j+=i)
			    primes[j] = true;
		    }
	    }

	int cPrime = 5;
	long sum = 0;
	for (int i = 6 ; i < primes.length ; i++)
	    {
		if (!primes[i])
		    {
			long matcher = match(cPrime,i);
			sum += matcher;
			cPrime = i;
			System.out.println((i+0.0)/primes.length);
		    }
	    }
	System.out.println(sum);
    }

}

File: problem134.py
from math import log10,pow
import sys

primes = [0]*101

for i in range(2,len(primes)):
    if primes[i] == 0:
        for j in range(i*2,len(primes),i):
            primes[j] = 1

def find(p1,p2):    
    mult = pow(10,1+int(log10(p1)))
    i = int(mult)
    while (i*mult+p1) % p2 != 0:
        i += 1
    return p1,i

primepair = [7,11]
print find(primepair[0],primepair[1])
for i in range(12,len(primes)):
    if primes[i] == 0:
        primepair[0] = primepair[1]
        primepair[1] = i
        print find(primepair[0],primepair[1])
